.env

```diff
@@ -7,3 +7,8 @@ DB_NAME="test"
 NODE_ENV="development"
 SECRET_KEY="secret"
 SESSION_SECRET='Kishan'
+
+EMAIL_ID = 'kishan.dobariya@bacancytechnology.com'
```

Comment:
change your id, place 'enter your id' insctead of kishan.dobariya@bacancytechnology.com